### DISCUSSION TITLE
[HW3] Reasonable accuracy Problem 2.2

### DISCUSSION THREAD

**Discussion by dinhkt** [seominjoon/kaist-ai605#142] [Q&A] [answered]
Hello professor and TAs,
I have questions in problem 2.2 and 2.3
1. In prob 2.2,I used a filtered vocab from Squad, which has 40000 words for training. First I tried to create noise by put a random character to a random position in word, and the accuracy of correction is around 10%. Then I create noise by randomly double a character in the word, and the accuracy is about 25-30%. How should noise be created and what is the expected accuracy? 
2. Problem 2.3 is still spelling correction for a word, but now consider the other words in the sentence, right? Can you give a hint how to design that?

**Comment by seominjoon** [comment, marked as answer]
1. What you tried is reasonable, and other possible ideas are: partial shuffing, missing letter, etc.
2. One way is to just use the same model as 2.2, but you will need a new data for training.

**Reply by dinhkt** [reply]
Thank you professor, 
For question 2, so that we can split the sentence into characters and space and then encode ?

**Reply by seominjoon** [reply]
Yes, and there is no single right answer.
